manifests/plugin/dns/route53.pp

```diff
@@ -0,0 +1,15 @@
+# @summary Install the Amazon Route53 DNS plugin for Foreman proxy
+#
+# @param aws_access_key
+#   The Access Key ID of the IAM account
+#
+# @param aws_secret_key
+#   The Secret Access Key of the IAM account
+#
+class foreman_proxy::plugin::dns::route53 (
+  String $aws_access_key = '', # lint:ignore:empty_string_assignment,
+  String $aws_secret_key = '', # lint:ignore:empty_string_assignment,
```

Comment:
You don't need to have a default. Especially with keys I think it can be ok not to have defaults.

Reply:
Thanks, I removed the default key variables being set to empty string.